### PR TITLE
AF Zoom-Nikkor 28-200mm and Nikkor Ai 24mm f/2.8

### PR DESCRIPTION
**Pull request by sarunasb**
Replacing AF Zoom-Nikkor 28-200mm f/3.5-5.6G IF-ED (full frame) with hopefully better calibration from newly submitted samples.

### CODE REVIEW

File: data/db/slr-nikon.xml [hunk 1]
```diff
@@ -3060,22 +3060,38 @@
     <lens>
         <maker>Nikon</maker>
         <model>Nikon AF Zoom-Nikkor 28-200mm f/3.5-5.6G IF-ED</model>
-        <model lang="en">Nikkor AF 28-200mm f/3.5-5.6G IF-ED</model>
```
Comment (bronger) [inline data/db/slr-nikon.xml]: Please restore this line.  The `<model>` line is used for automatic matching; therefore, I should be what exiv2 says.  In contrast, the `<model lang="en">` line is for the human eye.  For Nikon lenses, Lensfun removed “Nikon” and “Zoom” to make it more compact.

File: data/db/slr-nikon.xml [hunk 2]
```diff
@@ -3060,22 +3060,38 @@
...
+            <distortion model="ptlens" focal="125" a="0.00065" b="0.03125" c="-0.03952" />
+            <distortion model="ptlens" focal="200" a="0.01851" b="-0.05192" c="0.08089" />
+            <tca model="poly3" focal="28" vr="1.0002152" vb="1.0000092" />
+            <tca model="poly3" focal="38" vr="1.0001291" vb="1.0000999" />
+            <tca model="poly3" focal="48" vr="1.0000649" vb="1.0000876" />
+            <tca model="poly3" focal="62" vr="0.9999990" vb="1.0001145" />
+            <tca model="poly3" focal="85" vr="0.9999428" vb="1.0000457" />
+            <tca model="poly3" focal="125" vr="0.9999015" vb="0.9999823" />
+            <tca model="poly3" focal="200" vr="0.9999001" vb="0.9998229" />
+            <vignetting model="pa" focal="28" aperture="4.5" distance="10" k1="0.6082" k2="-2.2120" k3="1.3423" />
```
Comment (bronger) [inline data/db/slr-nikon.xml]: Please remove the vignetting data.  Vignetting images always must be made through a diffuser (e.g. paper or milk glass).  Also, remove the “vignetting” directory.

**Comment by sarunasb** [comment]
<model lang... > line restored; vignetting lines removed.

File: data/db/slr-nikon.xml [hunk 3]
```diff
@@ -3065,17 +3078,20 @@
...
-            <tca model="poly3" focal="28" vr="1.0002275" vb="1.0000944"/>
-            <tca model="poly3" focal="42" vr="1.0000854" vb="1.0000926"/>
-            <tca model="poly3" focal="50" vr="1.0000529" vb="1.0001110"/>
-            <tca model="poly3" focal="66" vr="0.9999807" vb="1.0001296"/>
-            <tca model="poly3" focal="95" vr="0.9999182" vb="1.0000201"/>
-            <tca model="poly3" focal="135" vr="0.9998602" vb="0.9999489"/>
-            <tca model="poly3" focal="200" vr="0.9999000" vb="0.9999004"/>
+            <distortion model="ptlens" focal="28" a="0.03547" b="-0.10778" c="0.08687" />
+            <distortion model="ptlens" focal="38" a="0.02089" b="-0.04522" c="0.02617" />
+            <distortion model="ptlens" focal="48" a="0.2973" b="0.67995" c="-0.12197" />
```
Comment (bronger) [inline data/db/slr-nikon.xml]: These values are way too high.  Please check them.

**Comment by bronger** [comment]
I merged this prematurely.  Please check the line commented by me.

File: data/db/slr-nikon.xml (hunk 3 above)
Comment (sarunasb) [inline data/db/slr-nikon.xml]: Sorry! It should have been:

`<distortion model="ptlens" focal="48" a="0.01934" b="-0.04754" c="0.04983" />`

Will make a pull request.

**Pull request merged**